src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -214,29 +227,44 @@ private bool ValidateRootBuildOptions()
         }
 
         /// <summary>
-        ///     Obtain the root build.
+        ///     Obtain the root builds to start with.
         /// </summary>
-        /// <returns>Root build to start with.</returns>
-        private async Task<Build> GetRootBuildAsync()
+        /// <returns>Root builds to start with, or null if a root build could not be found.</returns>
+        private async Task<IEnumerable<Build>> GetRootBuildsAsync()
         {
-            if (!ValidateRootBuildOptions())
+            if (!ValidateRootBuildsOptions())
             {
                 return null;
             }
 
             IRemote remote = RemoteFactory.GetBarOnlyRemote(_options, Logger);
 
             string repoUri = GetRepoUri();
+            List<int> rootBuildIds = new List<int>();
```

Comment:
I believe you can do `List<int> rootBuildIds = new List<int>(_options.RootBuildIds);` and then delete the `AddRange` below.